.github/workflows/merge-main.yml

```diff
@@ -52,8 +52,6 @@ jobs:
             -p NAME=${{ github.event.repository.name }}
             -p REACT_APP_USER_POOLS_ID=${{ vars.REACT_APP_USER_POOLS_ID }}
             -p REACT_APP_USER_POOLS_WEB_CLIENT_ID=${{ vars.REACT_APP_USER_POOLS_WEB_CLIENT_ID }}
-            -p REACT_APP_REDIRECT_SIGN_OUT="${{ vars.REACT_APP_REDIRECT_SIGN_OUT }}"
-            -p REACT_APP_AWS_DOMAIN=${{ vars.REACT_APP_AWS_DOMAIN }}
```

Comment:
`REACT_APP_AWS_DOMAIN` is referenced in these files still:
- docker-compose.yml
- frontend/openshift.deploy.yml
- frontend/src/aws-export.js

Why are we removing this one? It seems required.

Reply:
There are now defaults in frontend/src/aws-exports.js.  Providing these envars overrides those.